ament_index_python/test/test_ament_index_python.py

```diff
@@ -113,28 +103,27 @@ def test_resource():
 
     resource, prefix = get_resource('resource_type4', 'foo')
     assert resource == 'foo', 'Expected different content'
-    assert os.path.basename(prefix) == 'prefix1', 'Expected different prefix'
+    assert PurePath(prefix).name == 'prefix1', 'Expected different prefix'
```

Comment:
Why use `PurePath` over `Path` here (and below)?

Reply:
In these cases, PurePath is preferred because there is no need to use the filesystem functionality included in Path